mobile-security-framework/info.json

```diff
@@ -0,0 +1,701 @@
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the Hash of the Scan File.",
+          "description": "Specify the Hash of the Scan File."
+        }
+      ],
+      "output_schema": {}
+    },
+    {
+      "operation": "view_source_files",
+      "title": "View Source Files",
+      "annotation": "view_source_files",
+      "description": "View Resource file of the Scan result",
+      "category": "investigation",
+      "is_config_required": true,
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "file",
+          "title": "File Path",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the relative path of the Source File which you want to See. By default, Path directory is in /home/mobsf/.MobSF/{hash_key}/java_source",
+          "description": "Specify the relative path of the Source File which you want to See. By default, Path directory is in /home/mobsf/.MobSF/{hash_key}/java_source"
+        },
+        {
+          "name": "hash",
+          "title": "Hash",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the Hash of the Scan File",
+          "description": "Specify the Hash of the Scan File"
+        },
+        {
+          "name": "type",
+          "title": "File Type",
+          "type": "select",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the App file type.",
+          "description": "Specify the App file type.",
+          "options": [
+            "apk",
+            "ipa",
+            "studio",
+            "eclipse",
+            "ios"
+          ]
+        }
+      ],
+      "output_schema": {}
+    },
+    {
+      "operation": "compare_apps",
+      "title": "Compare Apps",
+      "annotation": "compare_apps",
+      "description": "Compare Scan Result of two different Scans",
+      "category": "investigation",
+      "is_config_required": true,
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "hash1",
+          "title": "Hash of First Scan",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the Hash of the First Scan File.",
+          "description": "Specify the Hash of the First Scan File."
+        },
+        {
+          "name": "hash2",
+          "title": "Hash of Second Scan",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the Hash of the Second Scan File.",
+          "description": "Specify the Hash of the Second Scan File."
+        }
+      ],
+      "output_schema": {}
```

Comment:
output schema missing?

Reply:
The outputs of the this actions are different for every file and are very large in size, So after discussion it's decided to leave it as blank